### ISSUE TITLE
Alternative toolbar

### ISSUE THREAD

**Issue by Gena1005** [inorichi/tachiyomi#2889]
### Why/User Benefit/User Problem
I want to offer my thought about improving the user experience in "Tachiyomi", and it consists in adding a function called "Alternative toolbar".  (I’ve painted how this work will work at the bottom ⬇️) Pros: The "Tachiyomi" interface will look "cleaner" and more minimalistic, plus instead of this useless button, it will be possible to make a "scrollable" text (Manga / Manhwa / Manhua name) It seems to me too  good idea 👍🏻😁
### What/Requirements
When you click on this option (Alternative toolbar), the "Back" button (<-) will disappear from the upper left screen.  Very important for those who use gestures from android 10, or a third-party gesture application, such as: Fluid Navigation Gestures
![Screenshot_20200418-081439852 (1)](https://user-images.githubusercontent.com/62114135/79624812-91b1c400-814e-11ea-9b31-c3f4a0269989.jpg)
![Screenshot_20200418-081544312 (1)](https://user-images.githubusercontent.com/62114135/79624826-adb56580-814e-11ea-8f89-da4b63009e2c.jpg)

**Comment by arkon** [comment]
I'm confused, is this requesting something new?

**Comment by Gena1005** [comment]
> Я в замешательстве, это запрашивает что-то новое?

Hello.  Yes, this is a function request, I’m sorry, I’ll correct it so that everything is clear to you, I apologize for the inconvenience

**Comment by arkon** [comment]
The button at the top is actually called an up button in Android, and doesn't always do the same thing as the navigation back button. For example, opening the chapters page directly from a notification should have the up button go to the library, but back to your previous app if you press back.

tl;dr we'll be keeping the up button to stay consistent with Android UX norms.
